### DISCUSSION TITLE
Media Chrome v1

### DISCUSSION THREAD

**Discussion by heff** [muxinc/media-chrome#410] [Proposals]
What does 1.0 need? Notes from office hours discussion:

### Reliability
- [x] More tests 70+%
- [x] Tests for the complex state management, both ways
- [ ] Tests for React?

### API Cleanup
- [x] Decide naming Attributes, all dashed or not all dashed
- [x] Decide if we use `media-` prefix on all the media state vars. Answer **No dash**
- [x] Naming CSS variables, consistency (finish & merge #473 )
- [x] Make all attribute names consistent in smushed case
- [ ] Document naming guidelines for CSS vars, elements, attributes/corresponding properties
- [x] Combine subtitles & captions

### Docs
- [ ] Guides - you’re trying to do X.
- [x] General cleanup on consistency of docs, e.g. documenting all attributes of components.
- [ ] Document all media state attributes in one place and then for each component list the media state attribute that are received

### Themes
- [x] Themes confidence. Not going to change underfoot.

### Custom media elements
- [x] Define the minimum required API for custom media elements. 
    - Aiming for min requirement is play/pause & whichever state and events are required for that.
    - [x] Update Media Controller to work with that minimum assumption
- [x] First-class pages for each media element

### Project usability
- [x] NPM modules: ESM only for 1.0
  - Can make it hybrid later

### Marketing/Design
- [x] Implement New Logo

### React

- [ ] Update React scripts so they work for experimental components & media theme

### Updates in the wild

- [x] media-chrome.org
- [ ] code sandboxes

### Features

**None required for 1.0. Trying to get them in and promoted out of experimental but they are not blockers for 1.0**

- [ ] Captions menu button
- [ ] Way to create a menu
- [ ] List
- [ ] Full on settings menu?
- [ ] Dialog feature from Mux Player?
- [ ] Error dialog and management?

**Comment by cjpillsbury** [comment]
Let's add "audit + cleanup of all of our examples" to this list.
